docs/docs/tutorials/orca_k8s/orca_k8s_tutorial.md

```diff
@@ -0,0 +1,563 @@
+
+```python
+from bigdl.orca import init_orca_context
+
+init_orca_context(cluster_mode, master, container_image, 
+                  cores, memory, num_nodes, driver_cores, driver_memory, 
+                  extra_python_lib, penv_archive, jars, conf)
+```
+
+In `init_orca_context`, you may specify necessary runtime configurations for running the example on K8s, including:
+* `cluster_mode`: a String that specifies the underlying cluster; valid value includes `"local"`, `"yarn-client"`, `"yarn-cluster"`, `"k8s-client"`, `"k8s-cluster"`, `"bigdl-submit"`, `"spark-submit"`, etc.
+* `master`: a URL format to specify the master address of K8s cluster.
+* `container_image`: a String that specifies the name of docker container image for executors.
+* `cores`: an Integer that specifies the number of cores for each executor (default to be `2`).
+* `memory`: a String that specifies the memory for each executor (default to be `"2g"`).
+* `num_nodes`: an Integer that specifies the number of executors (default to be `1`).
+* `driver_cores`: an Integer that specifies the number of cores for the driver node (default to be `4`).
+* `driver_memory`: a String that specifies the memory for the driver node (default to be `"1g"`).
+* `extra_python_lib`: a String that specifies the path to extra Python packages, one of `.py`, `.zip` or `.egg` files (default to be `None`).
+* `penv_archive`: a String that specifies the path to a packed Conda archive (default to be `None`).
+* `jars`: a String that specifies the path to needed jars files (default to be `None`).
+* `conf`: a Key-Value format to append extra conf for Spark (default to be `None`).
+
+__Note__: 
+* All arguments __except__ `cluster_mode` will be ignored when using `spark-submit` to submit and run Orca programs.
+
+After the Orca programs finished, you should call `stop_orca_context` at the end of the program to release resources and shutdown the underlying distributed runtime engine (such as Spark or Ray).
+```python
+from bigdl.orca import stop_orca_context
+
+stop_orca_context()
+```
+
+For more details, please see [OrcaContext](https://bigdl.readthedocs.io/en/latest/doc/Orca/Overview/orca-context.html).
+
+## 1.2 K8s Client&Cluster
+The difference between k8s-client and k8s-cluster is where you run your Spark driver. 
+
+For k8s-client, the Spark driver runs in the client process (outside the K8s cluster), while for k8s-cluster the Spark driver runs inside the K8s cluster.
+
+Please see more details in [K8s-Cluster](https://spark.apache.org/docs/latest/running-on-kubernetes.html#cluster-mode) and [K8s-Client](https://spark.apache.org/docs/latest/running-on-kubernetes.html#client-mode).
+
+
+## 1.3 Load Data from Network File Systems (NFS)
+When you are running programs on K8s, please load data from a network file system (NFS) instead of the local file system.
```

Comment:
better change to load from volumes and we use NFS in this tutorial as an example